assets/panel/debugger.properties

```diff
@@ -919,9 +919,10 @@ whyPaused.other=Debugger paused
 # keyboard shortcuts that use the control key
 ctrl=Ctrl
 
-# LOCALIZATION NOTE (anonymous): The text that is displayed when the
-# display name is null.
-anonymous=(anonymous)
+# LOCALIZATION NOTE (anonymous): this string is used to display JavaScript
```

Comment:
Need to update also the string reference here

Reply:
oops, done